site/en/fugu-showcase/fugu-showcase.njk

```diff
@@ -0,0 +1,245 @@
+{% endblock %}
+
+{% block content %}
+  {% if hero %}
+    <div class="display-flex justify-content-center lg:pad-left-400 lg:pad-right-400">
+      {% Img
+        class="hero-image object-fit-cover",
+        src=hero,
+        alt=alt,
+        width="960",
+        height="480",
+        sizes="(min-width: 960px) 960px, 100vw"
+      %}
+    </div>
+  {% endif %}
+  <div class="display-flex gap-top-300 lg:gap-top-400">
+    <div class="display-flex justify-content-center width-full">
+      <article class="width-full pad-left-400 pad-right-400">
+        <div class="fugu-container width-full center-margin pad-400">
+          <form class="search-apps-apis pad-top-400">
+            <div class="display-flex align-items-center">
+              <p class="search-input">
+                {{ icon('search', {hidden: false}) }}
+                <input class="search-apps search-box__input" list="available-apps" placeholder="Search apps and APIs" aria-label="Search apps and APIs" />
+              </p>
+              <p>
+                <label for="example-enhanced-select">FILTER BY: </label>
+                <enhanced-select class="search-apis" label="API" fixedLabel>
+                  <select name="example-enhanced-select">
+                    {% for availableAPI in fuguApis.availableApis %}
+                      <option value="{{ availableAPI }}">{{ availableAPI }}</option>
+                    {% endfor %}
+                  </select>
+                </enhanced-select>
+              </p>
+            </div>
+          </form>
+
+          <div class="pills pad-top-400"> 
+            {% from 'macros/icon.njk' import icon with context %}
+            {% for availableAPI in fuguApis.availableApis %}
+              <span class="tag-pill" hidden>
+                {{ availableAPI }}
+                <button class="button" aria-label="{{ 'i18n.common.close' | i18n(locale) }}">
+                  {{ icon('close') }}
+                </button>
+              </span>
+            {% endfor %}
+          </div>
+
+          <div class="project-grid pad-top-400">
+            {% for item in fuguApis.fuguItems %}
+              {% set apis = "" %}
+              {% for api in item.usedAPIs %}
+                {% set name = api.name | replace(" ", '-') %}
+                {% set apis = apis + " " + name %}
+              {% endfor %}
+                
+              {% set searchApis = "" %}
+              {% for api in item.usedAPIs %}
+                {% set searchApis = searchApis + " " + api.name %}
+              {% endfor %}
+
+              {% set apis = apis | trim | lower | replace("(", "") | replace(")", "") %}
+              <article id="{{ item.screenshot | replace(".webp", "") }}" data-name="{{ (item.title | lower) + " " + (searchApis | lower) + " " + (item.appURL | lower) }}" data-used-apis="{{ apis }}" class="blog-card rounded-lg pad-300 md:pad-400 width-full">
+                <h2 class="type--h5">
```

Comment:
There is *a lot* going on here at template level. Consider moving all that logic into a 11ty data file having it readily prepared when rendering the template.